src/components/Popup/Popup.js

```diff
@@ -45,7 +45,7 @@ const Popup = ({
 
   // Styleguidist try to load every file in the folder if we don't put index.js
   const PopupComponent = React.lazy(() =>
-    import(`../../popups/${componentName}/index.js`),
+    import(`../Popups/${componentName}/index.js`),
```

Comment:
@edellucien why did you move this directory? I would like to have `menus` and `popups` next to `components`.

Reply:
This has been decided to be moved in a intern ticket.

Reply:
Fixed